bdd/spec/032_json_spec.sh

```diff
@@ -0,0 +1,45 @@
+Include build_tools.sh
+
+Describe "JSON"
+  Describe "basic construction and printing"
+    before() {
+      sourceToAll "
+        from @std/app import start, print, exit
+        from @std/json import JSON, toJSON, toString, JSONBase, JSONNode, IsObject, Null
+
+        on start {
+          1.0.toJSON().print();
```

Comment:
This reads a bit odd, like a tuple in rust.

Reply:
Oh, but it's the constant `1.0` having the `toJSON` method called on it. :)

That you've got it a bit confused with tuples is part why I haven't implemented tuples; keeping it unambiguous in the grammar is complex for multiple reasons (tuple definition, tuple type definition, and even tuple accessor syntax could be confusing).

Reply:
Yep, I agree. No tuples!